testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -171,16 +171,65 @@ class TestContacts(object):
     def universe():
         return mda.Universe(PSF, DCD)
 
-    def _run_Contacts(self, universe, start=None, stop=None, step=None, **kwargs):
+    def _run_Contacts(
+        self, universe,
+        start=None, stop=None, step=None, **kwargs
+    ):
         acidic = universe.select_atoms(self.sel_acidic)
         basic = universe.select_atoms(self.sel_basic)
+
```

Comment:
Improving readability is appreciated but I also don't think it's worth affecting the git history for an empty line.
```suggestion
```